resources/nl-core/nl-core-NameInformation.xml

```diff
@@ -35,7 +33,100 @@
+        <expression value="(family.exists() or given.extension.where(url = 'http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier').where(value != 'IN').exists()) implies text.hasValue()" />
+      </constraint>
+    </element>
+    <element id="HumanName.text">
+      <path value="HumanName.text" />
+      <comment value="A representation of the full name, including given names and the familiy name. Although this element represents the official names, it is considered desirable to include unofficial names as well if commonly used, for example added in brackets.&#xD;&#xA;&#xD;&#xA;When implementing zib NameInformation, this rendition should include GivenName or Initials, the GivenName and the family name constructed according to the NameUsage concept." />
+      <condition value="nl-core-NameInformation-01" />
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.4" />
+        <comment value="FirstNames (implicit, main mapping is on .given)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.5" />
+        <comment value="Initials (implicit, main mapping is on .given)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.6" />
+        <comment value="GivenName (implicit, main mapping is on .given in a sibling element)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.7" />
+        <comment value="NameUsage (implicit, main mapping is on extension:nameUsage)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.11" />
+        <comment value="Prefix (implicit, main mapping is on .family.extension:prefix)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.10" />
+        <comment value="LastName (implicit, main mapping is on .family.extension:lastName)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.9" />
+        <comment value="PartnerPrefix (implicit, main mapping is on .family.extension:partnerPrefix)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nameinformation-v1.1-2020EN" />
+        <map value="NL-CM:20.4.8" />
+        <comment value="PartnerLastName (implicit, main mapping is on .family.extension:partnerLastName)" />
+      </mapping>
+    </element>
+    <element id="HumanName.family">
+      <path value="HumanName.family" />
+      <comment value="When implementing zib NameInformation, this rendition should contain a rendering of the Prefix, LastName, PartnerPrefix and PartnerLastName concepts, constructed according to the NameUsage concept." />

```

Comment:
Same as above, rendition is an unkown word to me. Just want to flag it. Ignore is if you think it is ok.

Reply:
Removed it